lms/djangoapps/mobile_api/groups/urls.py

```diff
@@ -22,10 +17,15 @@
         r'^invite/(?P<group_id>[\d]*)/members$',
         GroupsInvite.as_view(),               
         name='invite-to-group'
-    ),
+    ), 
     url(
-        r'^(?P<group_id>[\d]*)/members$',
-        GroupsMembers.as_view(),
-        name='members-in-group'
+        r'^delete/(?P<group_id>[\d]*)$',
```

Comment:
Does GroupsDelete need to be a separate view and endpoint? Can we use the same Django Rest Framework generic view (http://tomchristie.github.io/rest-framework-2-docs/api-guide/generic-views#retrieveupdatedestroyapiview) and override respective functions for REST operations? ie: someone makes a HTTP Delete to /groups/<groupid>  url --- and you can override the respective function in the Django view that it hits